### DISCUSSION TITLE
Unable to install headers  globally  from transmission11 github repo ?

### DISCUSSION THREAD

**Discussion by CodeBlocker52** [Cyfrin/security-and-auditing-full-course-s23#100] [General]
I tried to follow the installation steps as provided in the repo (https://github.com/transmissions11/headers?tab=readme-ov-file) to install  headers and use it in my 3-passwordstore-repo but it showing headers not found ?
I have already installed rust and cargo in my system , I am using WSL.

**Comment by johnsonstephan** [comment]
Can you share the exact steps you took and issue encountered so we can better support you. Thanks

**Reply by PatrickAlphaC** [reply]
What errors did you run into?
